inc/namespace.php

```diff
@@ -22,6 +22,9 @@ function bootstrap() {
 
 	// Load the Interest namespace.
 	Interest\bootstrap();
+
+	// Set the Vary headers.
+	add_action( 'init' , __NAMESPACE__ . '\\set_vary_headers' );
```

Comment:
```suggestion
	add_action( 'init', __NAMESPACE__ . '\\set_vary_headers' );
```

Addresses lint error, otherwise good to go!